tests/data/deltacode/scan_sorted01_new.json

```diff
@@ -86,7 +59,7 @@
       "is_script": false,
       "licenses": [
         {
-          "key": "apache-2.0",
+          "key": "apache-3.0",
```

Comment:
why is this changed to apache-3.0?

Reply:
@majurg I just wanted my factor list to be filled with "license change". Previously factors list was filled with status, so to make the test method work, I have to add other factors excluding status.

Reply:
@arnav-mandal1234 OK that makes sense.